robolectric-shadows/shadows-core/pom.xml

```diff
@@ -389,9 +427,29 @@
       </build>
       <dependencies>
         <dependency>
+          <groupId>com.android.tools.layoutlib</groupId>
```

Comment:
I didn't know these were available on Maven. Are they available for pre-21?

If not and they are in aosp, then we could consider building an android jar containing these classes.

see scripts/build_android.sh